### DISCUSSION TITLE
Kubernetes Deployment

### DISCUSSION THREAD

**Discussion by jsmueller7** [UCLALibrary/docker-cantaloupe#169] [Q&A]
Hello,

in your readme file I've read that you're running canteloupe on Kubernetes. Do you have a chart or something else I can get started with?

**Comment by ksclarke** [comment]
Hi @jsmueller7, a couple locations might help:

https://github.com/UCLALibrary/gitops_kubernetes/blob/main/apps/services-team-prod-environment-values.yaml#L137-L169
https://github.com/UCLALibrary/gitops_kubernetes/blob/main/apps/services-team-value-files/prod-cantaloupe.yaml
https://github.com/UCLALibrary/uclalib-helm-generic/tree/main/charts

Unfortunately, our systems person, who manages our kubernetes deployments for Cantaloupe, is out on family leave at the moment so our best source of information on how the whole process works isn't around to provide more detailed information.

We expect him back in two weeks though.

**Reply by jsmueller7** [reply]
Hi @ksclarke,
thank you for sharing your resources! This is actually helping me solve a problem right now. I'll probably find the rest myself!
